TrkReco/src/TrkRecoMcUtils_tool.cc

```diff
@@ -32,23 +32,23 @@
 namespace mu2e {
 
   class TrkRecoMcUtils : public mu2e::McUtilsToolBase {
```

Comment:
Mu2e software policy is that modules (and tools) must be upgraded to use validated configuration instead of parameter sets when modified.  Please update accordingly.

Reply:
done.